twilight-model/src/poll/mod.rs

```diff
@@ -0,0 +1,221 @@
+pub mod answer;
+pub mod answer_count;
+pub mod layout_type;
+pub mod media;
+pub mod results;
```

Comment:
We should not make these public since we re-export everything
```suggestion
mod answer;
mod answer_count;
mod layout_type;
mod media;
mod results;
```